lib/src/dynamics/world.dart

```diff
@@ -1004,12 +1005,15 @@ class WorldRayCastWrapper implements TreeRayCastCallback {
         ..setFrom(input.p1)
         ..scale(1 - fraction)
         ..add(_temp);
-      return callback.reportFixture(fixture, _point, _output.normal, fraction);
+      // TODO.0llie null-safety: Recheck class w/ spydon
+      return callback?.reportFixture(
+              fixture, _point, _output.normal, fraction) ??
```

Comment:
Calls breaking over multiple lines should have trailing commas, something like this:
```dart
return callback?.reportFixture(
              fixture, 
             _point,
             _output.normal,
             fraction,
           ) ?? ...
```
You can just add the comma and `dart format` should do the rest.

Reply:
And 0 looks like a good default